### PR TITLE
Issue 17

### LINKED ISSUE #17: Don't clone the RedFish emulator in the setup.sh script
The setup.sh script is checking out the original RedFish emulator and then copies all the files part of this repository in to the RedFish emulator codebase. 

However, the process replaces most of the files in the emulator except for a few missing ones like:
- api_emulator/version.py
- api_emulator/account_service.py
- api_emulator/resource_dictionary.py
- api_emulator/exceptions
- templates/*

The above files could be easily replicated in this codebase and we could remove the tedious copy-paste based installation/execution.

### PR DESCRIPTION
**Pull request by bmesnet**
Code proposed to answer #17 
Modified `setup.sh ` file to:

- Remove the Redfish-Interface-Emulator cloning
- Remove the creation of a specific working directory
- Move the default virtual environment creation to an option
- Added python3 release checking

Added the few missing files copying them from Redfish-Interface-Emulator github:

- api_emulator/version.py
- api_emulator/account_service.py
- api_emulator/resource_dictionary.py
- api_emulator/exceptions.py
- templates/*

Updated README.md

### CODE REVIEW

File: templates/browse.html
Comment (christian-pinto) [inline templates/browse.html:1]: If this file was taken from the DMTF Redfish emulator we should add their header. Just to make sure we give credit on whatever we re-use.

File: templates/index.html
Comment (christian-pinto) [inline templates/index.html:1]: Same as for the previous comment on "browse.html".

**Review by christian-pinto — CHANGES_REQUESTED** [review]
It is mostly about adding the proper license headers to some of the files. Otherwise it all looks good to me.

File: templates/browse.html
Comment (bmesnet) [reply templates/browse.html:1]: header added

File: templates/index.html
Comment (bmesnet) [reply templates/index.html:1]: header added

**Review by christian-pinto — APPROVED** [review]
Thanks @bmesnet looks good now

**Comment by christian-pinto** [comment]
This closes issue #17

**Comment by christian-pinto** [comment]
Closes #17

**Pull request merged**